gtsam/sfm/TranslationRecovery.h

```diff
@@ -99,10 +99,18 @@ class TranslationRecovery {
 
   /**
    * @brief Create random initial translations.
-   *
+   * 
+   * @param rng random number generator
+   * @return Values
+   */
+  Values initializeRandomly(std::mt19937 &rng) const;
```

Comment:
Actually, I think it should be `std::mt19937 *rng`: use a pointer for i/o, not a reference. This should also be fixed in all other cases, but let's not add another case.